core/src/main/java/tc/oc/pgm/observers/tools/VisibilityTool.java

```diff
@@ -18,6 +18,12 @@
 
 public class VisibilityTool implements InventoryMenuItem {
 
+  final InventoryMenu in;
+
+  public VisibilityTool(InventoryMenu in) {
+    this.in = in;
```

Comment:
```suggestion
  final InventoryMenu menu;

  public VisibilityTool(InventoryMenu menu) {
    this.menu = menu;
```

Reply:
Wops this is just code I forgot to remove